lib/backend-api/src/client.rs

```diff
@@ -6,6 +6,16 @@ use anyhow::{bail, Context as _};
 use cynic::{http::CynicReqwestError, GraphQlResponse, Operation};
 use url::Url;
 
+pub fn get_proxy() -> Result<Option<reqwest::Proxy>, anyhow::Error> {
+    if let Ok(scheme) = std::env::var("http_proxy").or_else(|_| std::env::var("HTTP_PROXY")) {
+        let proxy = reqwest::Proxy::all(scheme)?;
+
+        Ok(Some(proxy))
+    } else {
+        Ok(None)
+    }
+}
```

Comment:
I believe we already have this somewhere (CLI?). We should reuse it, not copy&paste

Reply:
We can move it to the api crate, and re-use it in the CLI.

But I believe it might be in the registry crate ATM...

Reply:
but `backend-api` does not depend on `api`. It'll be a dependency just for a single function.